cfpb/design-manual
# Recompile assets to fix notification icons

We accidentally lost some of the icon CSS updates. This fixes that by recompiling the assets.

## Testing

1. Pull branch
1. `npm start`
1. Visit http://localhost:4000/page-components/form-fields.html#states-1
1. Observe correctly positioned icons.

## Screenshots

### Before
![screen shot 2018-04-10 at 17 47 06](https://user-images.githubusercontent.com/1044670/38585302-60d73a52-3ce7-11e8-9ed5-3c3ff44ad0bf.png)

### After
![screen shot 2018-04-10 at 17 47 56](https://user-images.githubusercontent.com/1044670/38585312-6634b11e-3ce7-11e8-9e04-5de0a11eca3c.png)

## Checklist
